rebasehelper/specfile.py

```diff
@@ -147,13 +146,49 @@ def __init__(self, content):
     def __str__(self):
         """Join SPEC file sections back together."""
         content = []
-        for header, section in six.iteritems(self.sections):
+        for header, section in self.sections:
             if header != '%package':
                 content.append(header + '\n')
             for line in section:
                 content.append(line + '\n')
         return ''.join(content)
 
+    def section(self, name):
```

Comment:
Shouldn't this rather be named `get_section`?

Reply:
It certainly could, but it is supposed to be a drop-in replacement for the current `spec_content.section[NAME]`.

Reply:
You are right that, moreover `get_section` would be a bit too long with our use-case in mind